src/components/themed/WalletListCreateRow.js

```diff
@@ -47,7 +49,7 @@ export const WalletListCreateRowComponent = (props: WalletListCreateRowProps) =>
     if (walletType != null) {
       dispatch(createAndSelectWallet({ walletType })).then(handleRes)
     } else if (pluginId != null) {
-      dispatch(createAndSelectToken({ currencyCode, pluginId })).then(handleRes)
+      dispatch(createAndSelectToken({ tokenCode: currencyCode, pluginId, createWalletId })).then(handleRes)
```

Comment:
Let keep this as `currencyCode`. Consistent naming makes the code easier to work with for everyone, and this is the term we use all through the app to indicate an exchange symbol like "BTC" or "ETH" or "DAI". It doesn't necessarily have to be a token.

Otherwise, if you really want to fix it, you could upgrade this file to use `tokenId`, which is the less ambiguous way to identify things based on contract address.

But yeah, we want to keep the naming consistent on this legacy stuff so it's more obvious where fixes are needed.

Reply:
I made this change because the bug came about due to ambiguity of currencyCode vs token(Currency)Code but I can revert

Reply:
On second thought, searching for currencyCode would still show this line. Regarding tokenId upgrade, we are already tracking that here: https://app.asana.com/0/1200895675496964/1202827562867788/f

I think there's value to keeping this here in case further changes are made before getting to that task